customer-account-extension/src/CustomerAccountExtension.liquid

```diff
@@ -1,25 +1,35 @@
 {%- if flavor contains "react" -%}
   import React from 'react';
-  import {render, Banner} from '@shopify/customer-account-ui-extensions-react';
-  
-  render('customer-account.dynamic.render', (api) => <App api={api} />);
-  
-  function App(api) {
-    return <Banner>{api.i18n.translate("welcome")}</Banner>;
+  import { Banner } from '@shopify/ui-extensions-react/customer-account';
+  import{ reactExtension, useTranslate} from '@shopify/ui-extensions-react/customer-account';
+
+  export default reactExtension(
+    'customer-account.dynamic.render',
+     () => <CustomerAccountExtension />
+  );
+
+  function CustomerAccountExtension() {
+    const translate = useTranslate();
+
+    return (
+      <Banner title="{{ name }}">
```

Comment:
will this be replaced with shop name?

Reply:
No. Should be replaced by app name

Reply:
not by us right? but liquid will fill it automatically?

Reply:
Yes, you are right.